src/lib/y2storage/filesystems/type.rb

```diff
@@ -30,6 +30,82 @@ class Type
       include StorageEnumWrapper
 
       wrap_enum "FsType"
+
+      FILESYSTEMS = {
```

Comment:
nitpicking: not sure if I like the name. What about `PROPERTIES` or something similar?